src/bootstrap/src/utils/render_tests.rs

```diff
@@ -185,58 +187,58 @@ impl<'a> Renderer<'a> {
     }
 
     fn render_test_outcome_verbose(&self, outcome: Outcome<'_>, test: &TestOutcome) {
-        print!("test {} ... ", test.name);
-        self.builder.colored_stdout(|stdout| outcome.write_long(stdout)).unwrap();
+        eprint!("test {} ... ", test.name);
+        self.builder.colored_stderr(|stdout| outcome.write_long(stdout)).unwrap();
```

Comment:
I changed instances of `colored_stdout` to `colored_stderr`, since that seems to be correct. Tests *look* okay for me running locally

Reply:
I'll do some more extensive manual testing later today

Reply:
This change looks good to me locally.